help/destinations/catalog/crm/salesforce.md

```diff
@@ -171,51 +186,55 @@ To correctly send your audience data from Adobe Experience Platform to the [!DNL
 
     * Add the following mapping between your XDM profile schema and your [!DNL Salesforce CRM] instance:
 
-    |XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory |
-    |---|---|---|
-    |`crmID`|`SalesforceId`|Yes|
+        |XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory |
+        |---|---|---|
+        |`crmID`|`SalesforceId`|Yes|
 
     * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you have defined in the **[!UICONTROL Attribute name]** field. Refer to the [[!DNL Salesforce CRM] documentation](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5) for guidance on supported attributes.
     ![Platform UI screenshot showing Target mapping for LastName.](../../assets/catalog/crm/salesforce/target-mapping-lastname.png)
 
+    #### Working with Contacts
```

Comment:
I believe that validation will fail if we indent a heading. Please don't indent headings or use `**bold**`.